### ISSUE TITLE
ember-token-auth and ember-oauth2 - the same thing?

### ISSUE THREAD

**Issue by sparkrevolutions** [amkirwan/ember-oauth2#27]
Reading through the documentation and being quite new to oAuth in general, I think its very confusing if these two addons are actually the same thing.

The documentation telling you to refer to the other addon continuously isn't great...

Attempting to install ember-oauth2 resulted in a error, so I chose ember-token-auth.

Are they the same? If not, what is the difference and which one should you use when developing using Ember CLI.

Thanks in advance

**Comment by mshamma** [comment]
Agreed.  I appreciate the effort on this project and I know you may be busy with other work, but the docs do lack a working example that (perhaps) incorporates where each framework begins and ends.
